apps/condo/package.json

```diff
@@ -71,6 +73,7 @@
     "jspdf": "^2.3.1",
     "less": "^3.12.2",
     "lodash": "^4.17.21",
+    "luxon": "^2.0.2",
```

Comment:
Right now we have 3 date libraries

Luxon, Moment and date-fns.

Reply:
Why cant we simply use date-fns here and not add new depenency?

Reply:
will remove moment and date-fns later. Use №1037 task from JIRA to see current status